lib/rules/lines-between-class-members.js

```diff
@@ -159,20 +232,34 @@ module.exports = {
                     const hasTokenInPadding = hasTokenOrCommentBetween(beforePadding, afterPadding);
                     const curLineLastToken = findLastConsecutiveTokenAfter(curLast, nextFirst, 0);
 
-                    if ((options[0] === "always" && !skip && !isPadded) ||
-                        (options[0] === "never" && isPadded)) {
-                        context.report({
-                            node: body[i + 1],
-                            messageId: isPadded ? "never" : "always",
-                            fix(fixer) {
-                                if (hasTokenInPadding) {
-                                    return null;
+                    if (configureList.length > 0) {
```

Comment:
This condition seems redundant since `configureList` is always a non-empty array?

Reply:
Yes, removed.